### DISCUSSION TITLE
No connection to webdav

### DISCUSSION THREAD

**Discussion by stepstra** [sircharlo/meeting-media-manager#2176] [Q&A] [answered]
Hi. For two or three weeks now, M3 can't connect to our shared /M3 media folder we use in 4shared. Is there any known issue?

**Comment by mtdvlpr** [comment]
Hi @stepstra, I will check it out when I have some time. 4shared has been known to cause issues, though.

**Reply by stepstra** [reply]
Did you find time to check on this? I recently opened an account with Koofr, but this says "Too many requests to the server. Please try again later." Doesn't seem to work either. :-(

**Reply by mtdvlpr** [reply]
https://forum.odrive.com/t/problem-with-4shared-account-and-webdav-protocol-unable-to-download-or-synchronize/9707

**Reply by mtdvlpr** [reply]
@stepstra, sorry for the late response! I've been really busy lately. Koofr is working for us and has been working flawlessly for years. I would give that another go. Did you setup a subfolder for Koofr instead of using the root folder? That could be an issue.

**Reply by mtdvlpr** [reply]
I also get an Unauthorized error when trying to connect to 4shared. Even when trying to loging to webdav.4shared.com directly (without M3).

**Reply by mtdvlpr** [reply]
For Koofr, you also need to setup a special password, specifically for M3 and not use your own profile's password.

**Reply by e-Feitosa** [reply]
You must have followed these steps to **generate the APP PASSWORD**, if not, you need to create a password. 
Note that when creating the password, if you copy and paste, there is a space between the password blocks (123r m7y1...), you must delete these spaces (**123rm7y1od0qqqd9**) leaving it as per the model
⬇️
![image](https://github.com/sircharlo/meeting-media-manager/assets/90984283/acf177ac-ed50-4334-be49-69e7088acba1)

**Reply by stepstra** [reply]
Thanks for the detailed help to setup the Koofr account. That helped a lot. (Man, that's complicated ;-))

**Comment by sircharlo** [comment]
This is the reason why eventually we'll move away from WebDAV syncing in the app itself, in favor of watching a folder (managed by your own cloud sync solution) instead.
That way, we'll avoid these intermittent issues.

**Reply by e-Feitosa** [reply]
@sircharlo, Speaking of this subject, some brothers ask me if there is any way to configure an application (Google Drive, OneDrive, etc.) on their cell phone to send media to the server and synchronize on SR, many do not have a PC / Laptop at home and cannot send the media in advance for the M3. I personally use the Koofr server and have 99% excellence.

**Comment by mtdvlpr** [comment, marked as answer]
Outdated as of `v24.10.6`
